packages/dappmanager/src/modules/stakerConfig/setStakerConfig.ts

```diff
@@ -125,6 +127,16 @@ export async function setStakerConfig<T extends Network>({
       `Web3signer version ${currentWeb3signerPkg.version} is lower than the minimum version ${web3signer.minVersion} required to work with the stakers UI. Update it to continue.`
     );
 
+  // If consensus client is lodestar, check that all the execution client and signer versions
+  if (stakerConfig.consensusClient?.dnpName.includes("lodestar")) {
```

Comment:
I would add to this check that the execution client and signer exist

Reply:
Okay, I can add this